<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gameStart</class>
 <widget class="QDialog" name="gameStart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>817</x>
     <y>0</y>
     <width>45</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_money_p">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>30</y>
     <width>45</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_money">
   <property name="geometry">
    <rect>
     <x>645</x>
     <y>35</y>
     <width>45</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TextLal</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_health_p">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>10</y>
     <width>45</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_health">
   <property name="geometry">
    <rect>
     <x>645</x>
     <y>10</y>
     <width>45</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>695</x>
     <y>10</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="value">
    <number>33</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="backcolor">
   <property name="geometry">
    <rect>
     <x>605</x>
     <y>5</y>
     <width>45</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_resource2_p">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>60</y>
     <width>45</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_resource2">
   <property name="geometry">
    <rect>
     <x>645</x>
     <y>60</y>
     <width>45</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <zorder>backcolor</zorder>
  <zorder>label_health_p</zorder>
  <zorder>label</zorder>
  <zorder>label_money_p</zorder>
  <zorder>label_money</zorder>
  <zorder>label_health</zorder>
  <zorder>progressBar</zorder>
  <zorder>label_resource2_p</zorder>
  <zorder>label_resource2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
